handlers.go

```diff
@@ -3600,6 +3604,67 @@ func (s *Server) handleDeleteCollection(c echo.Context, u *User) error {
 	return c.NoContent(http.StatusOK)
 }
 
+// handleDeleteContentFromCollection godoc
+// @Summary      Deletes a content from a collection
+// @Description  This endpoint is used to delete an existing content from an existing collection. If two or more files with the same contentid exist in the collection, delete the one in the specified path
+// @Tags         collections
+// @Param        coluuid path string true "Collection ID"
+// @Param        contentid path string true "Content ID"
+// @Param        path query string false "Path in which delete files (if found)"
+// @Produce      json
+// @Success      200  {object}  string
+// @Failure      400  {object}  util.HttpError
+// @Router       /collections/{coluuid}/content/{contentid} [delete]
+func (s *Server) handleDeleteContentFromCollection(c echo.Context, u *util.User) error {
+	coluuid := c.Param("coluuid")
+	contentid := c.Param("contentid")
+	path := c.QueryParam("path")
```

Comment:
I thought having the path in the address bar could pose some problems?

Reply:
We have it on other places throughout the API so I thought being consistent is better in the end, but I'll put it on the body following your suggestion